data.json

```diff
@@ -856,11 +856,17 @@
     "githubUsername": "SulthanNK",
     "imageUrl": "https://user-images.githubusercontent.com/32130871/117547009-e4586980-b04a-11eb-8daa-baad5ffb4949.png",
     "issueNumber": 482
-  },
+  },  
```

Comment:
What is changed here?

Reply:
ok now I see it, please remove the spaces here this will cause an issue with prettier linter

Reply:
Yeah I found out that there are some spaces.
This will cause a prettier error. I've suggested changes here https://github.com/EddieHubCommunity/awesome-github-profiles/pull/489#discussion_r629463289 you can directly accept them or modify the code by yourself it's up to you 🚀